src/Components/Admin/Rooms/rooms.js

```diff
@@ -15,14 +17,64 @@ const useStyles = makeStyles((theme) => ({
 
 export default function EscapeRooms() {
     const classes = useStyles();
+    const rooms = [
+        {
+            id: 1,
+            name: "Ahmed's Escape Room",
+        },
+        {
+            id: 2,
+            name: "Jay's Escape Room",
+        },
+        {
+            id: 3,
+            name: "Dhruvin's Escape Room",
+        },
+        {
+            id: 4,
+            name: "Amolik's Escape Room",
+        },
+        {
+            id: 5,
+            name: "Aaron's Escape Room",
+        },
+        {
+            id: 6,
+            name: "Vivian's Escape Room",
+        },
+        {
+            id: 7,
+            name: "Kevin's Escape Room",
+        },
+        {
+            id: 8,
+            name: "Kouthar's Escape Room",
+        },
+        {
+            id: 9,
+            name: "Jack's Escape Room",
+        },
+    ];
 
     return (
-        <Container component={"main"} maxWidth="xs">
+        <Container component={"main"} maxWidth="lg">
             <CssBaseline />
             <div className={classes.page}>
                 <Typography component="h1" variant="h5">
                     Escape Rooms Page
                 </Typography>
+                <Grid
+                    container
+                    item
+                    xs={12}
+                    spacing={1}
+                    alignItems="center"
+                    justify="flex-start"
+                >
+                    {rooms.map(function (room) {
+                        return <RoomCard key={room.id} data={room} />;
+                    })}
```

Comment:
```suggestion
                    {rooms.map((room) =>  (
                        <RoomCard key={room.id} data={room} />;
                    ))}
```
I am a fan of arrow functions. Just looks a lot cleaner and modern (and I believe better practice). Thoughts?

Reply:
I heard the opposite, at my current company (not going to mention by name on github publicly), we do this. Adding a level of nesting with a return statement is not preferred.

Reply:
Interesting didn't know that. Let's go with your way then.